CHANGELOG.md

```diff
@@ -84,6 +84,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fixed compression support for h2c protocol ([#4944](https://github.com/opensearch-project/OpenSearch/pull/4944))
 - Reject bulk requests with invalid actions ([#5299](https://github.com/opensearch-project/OpenSearch/issues/5299))
 - Support OpenSSL Provider with default Netty allocator ([#5460](https://github.com/opensearch-project/OpenSearch/pull/5460))
+- Improve tests.config property thread safety ([#5645](https://github.com/opensearch-project/OpenSearch/pull/5645))
```

Comment:
Generally, [test-related changes](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#changelog) don't need to go into the the changelog, so this can be omitted.